dashboard/app/models/contact_rollups_final.rb

```diff
@@ -15,4 +15,27 @@
 
 class ContactRollupsFinal < ApplicationRecord
   self.table_name = 'contact_rollups_final'
+
+  def self.truncate_and_overwrite
+    truncate
+    overwrite_from_processed_table
+  end
+
+  def self.truncate
+    truncate_sql = <<~SQL
+      TRUNCATE contact_rollups_final;
+    SQL
+
+    ActiveRecord::Base.connection.exec_query(truncate_sql)
```

Comment:
There is an [ActiveRecord `truncate` method](https://github.com/rails/rails/blob/v5.0.7.2/activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb#L332-L334) on the connection object.  It's pretty much exactly equivalent to what you've done here, except that it's more concise, and it'd be more flexible in the extremely unlikely event we ever had to change database adapters.

```rb
ActiveRecord::Base.connection.truncate('contact_rollups_final')
```